vespa-cloud/document-processing/README.md

```diff
@@ -21,115 +21,139 @@ and the asynchronous [Document API](https://docs.vespa.ai/en/document-api-guide.
 
 
-## Steps
 
-**Create a Vespa Cloud application:**
+## Steps
 
-Use _Create application_ at [console.vespa.ai](http://console.vespa.ai),
-choose a name like _document-processing_.
-This requires a Google or GitHub account,
-and will start your free trial if you don't already have a Vespa Cloud tenant.
+### Install the Vespa CLI:
+Using [Homebrew](https://brew.sh/):
+```
+$ brew install vespa-cli
+```
+You can also [download Vespa CLI](https://github.com/vespa-engine/vespa/releases)
+for Windows, Linux and macOS.
 
 
-**Clone sample apps:**
 
-<pre data-test="exec">
-$ git clone --depth 1 https://github.com/vespa-engine/sample-apps.git
-$ cd sample-apps/vespa-cloud/document-processing
-</pre>
+###Create your application in the Vespa Cloud:
+If you don't already have a Vespa Cloud tenant,
+create one at [console.vespa.ai](http://console.vespa.ai).
+This requires a Google or GitHub account, and will start your [free trial](https://cloud.vespa.ai/pricing#free-trial).
 
 
-**Create a self-signed certificate:**
 
-This certificate/key pair will be used to access the application deployed to Vespa Cloud.
+<!-- Use a distinct app name to avoid key collisions -->
+### Initialize `myapp-docproc/`
+Initialize `myapp-docproc/` to a copy of a
+[sample application package](https://docs.vespa.ai/en/cloudconfig/application-packages.html):
 <pre data-test="exec">
-$ openssl req -x509 -nodes -days 14 -newkey rsa:4096 \
--subj "/CN=cloud.vespa.example" \
--keyout data-plane-private-key.pem -out data-plane-public-cert.pem
-$ mkdir -p src/main/application/security
-$ cp data-plane-public-cert.pem src/main/application/security/clients.pem
+$ vespa clone vespa-cloud/document-processing myapp-docproc
+$ cd myapp-docproc
 </pre>
 
 
-**Create an API key:**
 
-In [console.vespa.ai](http://console.vespa.ai),
-choose _tenant_ and click _Keys_ to generate and save the _user API key_.
-The key is saved to `$HOME/Downloads/USER.TENANTNAME.pem`.
+### Let pom.xml specify the right tenant and application name:
+Set the properties `tenant` and `application` properties in pom.xml
+to your tenant and application name ("myapp-docproc")
 
 
-**Set tenant and application name in pom.xml:**
 
-Set the `tenant` and `application` names used when creating the application above.
+### Tell the Vespa CLI to use Vespa Cloud, with your application:
+```
+$ vespa config set target cloud
+$ vespa config set application <tenant-name>.myapp-docproc.default
+```
+Use the tenant and application name from step 2.
+
+<!-- Override VESPA_CLI_HOME to work around container filesystem limitations and set application used for automatic testing -->
+<!-- $PWD is set to $SD_DIND_SHARE_PATH by screwdriver.yaml - a special Docker-in-Docker path -->
+<pre data-test="exec" style="display:none">
+$ export VESPA_CLI_HOME=$PWD/.vespa TMPDIR=$PWD/.tmp
+$ mkdir -p $TMPDIR
+$ vespa config set target cloud
+$ vespa config set application vespa-team.document-processing.my-instance
+</pre>
```

Comment:
Is there a way we can hide this? It shows up for me when displaying the rendered readme: https://github.com/vespa-engine/sample-apps/blob/5910ec99aba9aece06dc52e74be64f50736ffbb9/vespa-cloud/document-processing/README.md